src/geotemporal_data.py

```
@@ -180,13 +185,21 @@ def point(self, lat: float, lon: float, snap_to_grid: bool = True) -> "Geotempor
-                data = self.data.sel(latitude=lat, longitude=lon, method="nearest", tolerance=10e-5)
+                data = self.data.sel(
+                    latitude=latitude,
+                    longitude=longitude,
+                    method="nearest",
+                    tolerance=10e-5,
+                )
             except KeyError:
-                raise errors.NoDataFoundError("User requested not to snap_to_grid, but exact coord not in dataset")
+                raise errors.NoDataFoundError(
+                    "User requested not to snap_to_grid, but exact coord not in dataset"
+                )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)